Description: Fix empty YAML_CPP_LIBRARIES
 The new version does not set EXPORT_TARGETS. As the value should be yaml-cpp
 anyhow, just set it directly.
Author: Jochen Sprickerhof <jspricke@debian.org>
Bug: https://github.com/jbeder/yaml-cpp/issues/774
Bug-Debian: https://bugs.debian.org/999594
Forwarded: https://github.com/jbeder/yaml-cpp/pull/1037
Last-Update: 2021-11-13

--- yaml-cpp-0.7.0+dfsg.orig/yaml-cpp-config.cmake.in
+++ yaml-cpp-0.7.0+dfsg/yaml-cpp-config.cmake.in
@@ -11,4 +11,4 @@ set(YAML_CPP_INCLUDE_DIR "@INCLUDE_INSTA
 include("${YAML_CPP_CMAKE_DIR}/yaml-cpp-targets.cmake")
 
 # These are IMPORTED targets created by yaml-cpp-targets.cmake
-set(YAML_CPP_LIBRARIES "@EXPORT_TARGETS@")
+set(YAML_CPP_LIBRARIES "yaml-cpp")
